Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/ListViewExtensions.SmoothScrollIntoView.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Microsoft.Toolkit.Uwp.UI
+{
+    /// <summary>
+    /// Smooth scroll the list to bring specified item into view
+    /// </summary>
+    public static partial class ListViewExtensions
+    {
+        /// <summary>
+        /// Smooth scrolling the list to bring the specified index into view
+        /// </summary>
+        /// <param name="listViewBase">List to scroll</param>
+        /// <param name="index">The index to bring into view. Index can be negative.</param>
+        /// <param name="itemPlacement">Set the item placement after scrolling</param>
+        /// <param name="disableAnimation">Set true to disable animation</param>
+        /// <param name="scrollIfVisible">Set false to disable scrolling when the corresponding item is in view</param>
+        /// <param name="additionalHorizontalOffset">Adds additional horizontal offset</param>
+        /// <param name="additionalVerticalOffset">Adds additional vertical offset</param>
+        /// <returns>Note: Even though this return <see cref="Task"/>, it will not wait until the scrolling completes</returns>
+        public static async Task SmoothScrollIntoViewWithIndex(this ListViewBase listViewBase, int index, ItemPlacement itemPlacement = ItemPlacement.Default, bool disableAnimation = false, bool scrollIfVisible = true, int additionalHorizontalOffset = 0, int additionalVerticalOffset = 0)
```

Comment:
These are async methods, so we should probably end them with `Async` naming convention.

Reply:
Done.